cfgov/jinja2/v1/_includes/organisms/_vars-mega-menu.html

```diff
@@ -135,14 +135,14 @@
     'title': 'Populations Served',
     'nav_items': [
         {'text': 'Financial Education for Adults', 'url': '/adult-financial-education/'},
-        {'text': 'Financial Education for Youth', 'url': '/practitioner-resources/youth-financial-education/'},
-        {'text': 'Low-to-Moderate Income', 'url': '/empowerment/'}
+        {'text': 'Financial Education for Youth', 'url': '/practitioner-resources/youth-financial-education/'}
```

Comment:
Not a blocker if people are hot to trot on this one, but you could leave the comma at the end of this line and then it wouldn't have to change.